downloads/index.md

```diff
@@ -3,4 +3,138 @@ layout: custom
+                            </div>
+                        </div>
+                        <div class="row center-xs">
+                            <div class="col-xs-6 col-sm-4">
+                                <a href="https://cakewallet.io"><img style="height: 115px;"  src="/img/cakewallet.png" alt="Cake Wallet Logo"></a>
+                            </div>
+                            <div class="col-xs-6 col-sm-4">
+                                <a href="https://monerujo.io"><img style="height: 115px;" src="/img/Monerujo-wallet.png" alt="Monerujo Logo"></a>
+                            </div>
+                            <div class="col-xs-12 col-sm-4">
+                                <a href="https://mymonero.com"><img src="/img/mymonero.png" alt="MyMonero Logo"></a>
+                            </div>
+                        </div>
+                    {% elsif data_downloads.gui_hash == nil and data_downloads.cli_hash != nil %}
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <h4 id="{{ data_downloads.platform | slugify }}">
+                                     <a href="//downloads.getmonero.org/cli/{{ data_downloads.cli_url }}"> {{ data_downloads.platform }} {% t downloads.clionly %}</a>
+                                </h4>
+                            </div>
+                         </div>
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <p><strong>SHA256 Hash:</strong></p> 
+                                <p class="hash"> {{ data_downloads.cli_hash }}</p>
+                            </div>
+                        </div>
+                    {% elsif data_downloads.gui_hash != nil and data_downloads.cli_hash == nil %}
+                        <div class="row">
+                            <h4 id="{{ data_downloads.platform | slugify }}">
+                                <a href="//downloads.getmonero.org/gui/{{ data_downloads.gui_url }}">{{ data_downloads.platform }}</a>
+                            </h4>
+                        </div>
+                        <div class="row">
+                            <p><strong>SHA256 Hash:</strong></p>
+                            <p class="hash"> {{ data_downloads.gui_hash }}</p>
+                        </div>
+                    {% elsif data_downloads.gui_hash != nil and data_downloads.cli_hash != nil %}
+                        <div class="row start-md">
+                            <div class="col-md-6 col-sm-12" >
+                                <h4 id="{{ data_downloads.platform | slugify }}">
+                                    <a href="//downloads.getmonero.org/gui/{{ data_downloads.gui_url }}">{{ data_downloads.platform }}</a>
+                                </h4>
+                                <p><strong>SHA256 Hash (GUI):</strong></p>
```

Comment:
GUI version of above, a replacement of this `SHA256 Hash (GUI):` (and all subsequent occurence) with `{% t downloads.hashgui %}` would be great. to handle separately because in french, the semicolon must have a leading space `...(GUI) :`